### PR TITLE
chore(experiments): rename A/B testing -> Experiments

### PR DESCRIPTION
**Pull request by jurajmajerik**
## Changes

We were [talking](https://posthog.slack.com/archives/C034XD440RK/p1721041492439889?thread_ts=1721041209.070149&cid=C034XD440RK) about possibly waiting until we do more of the "separate out Experiments" work, but I don't see a reason not do to it now:
- Experiments is the only viable name I can think of, and competing tools are called the same
- "A/B testing" is actually incorrect since we do offer more than 2 variants
- We already call it Experiments in other places, so this will make it consistent

posthog.com changes to follow

## How did you test this code?
👀

### CODE REVIEW

**Comment by jurajmajerik** [comment]
Interesting, I didn't know! Would be good to see that discussion and re evaluate if those reasons still make sense.

**Comment by corywatilo** [comment]
Found it! https://github.com/PostHog/product-internal/issues/459

There was also another [subsequent attempt](https://github.com/PostHog/posthog.com/pull/5605) to rename again last year.

**Comment by annikaschmid** [comment]
The original discussion is [here](https://github.com/PostHog/product-internal/issues/459).

James proposed to rename it to 'Experimentation' a while ago [here](https://posthog.slack.com/archives/C034XD440RK/p1713896900097539).

I was actually looking into this this week as part of my pricing / separate product research because I knew this would come up again, tl;dr almost all other companies call their product 'Experimentation'. [Overview here](https://posthog.slack.com/archives/C034XD440RK/p1713896900097539).

In terms of SEO, we, like every other company out there, are using the terms A/B testing and Experiments/Experimentation interchangeably, so I think it wouldn't hurt our SEO either way.

I prefer 'Experiments' over 'A/B Testing', because it sounds more like a serious product instead of a use case, especially once we split it out from feature flags. And I think a data scientist would say they run an 'experiment', not an 'a/b test'.

I prefer 'Experiments' over 'Experimentation', because it's an actual word you would use in a spoken sentence, vs 'Experimentation'.

**Comment by corywatilo** [comment]
There's some more relevant discussion here: https://github.com/PostHog/product-internal/issues/459 - though this isn't quite what I'm looking for.

There was also another [subsequent attempt](https://github.com/PostHog/posthog.com/pull/5605) to rename again last year.

**Comment by jurajmajerik** [comment]
I don't have any data to back this up, but I completely agree with Annika: "Experiments" feels like a more serious product name. It might just be my impression after checking out some competing products recently.

Last year's discussion didn't mention Statsig and Eppo, which I see as the most serious competitors right now, and both of them use "Experiments."

**Comment by andyvan-ph** [comment]
To answer's @jurajmajerik's question in Slack, there's no meaningful SEO impact. We can optimize for both quite easily and we'll probably still use both depending on context.

Fwiw, I like Experiments – it's way better than Experimentation, which was way too wordy.

**Comment by annikaschmid** [comment]
@jurajmajerik I briefly chatted about this in the growth review with James and Phani. There are no blockers from either of them, "Experiments" works for them. Let's 🚢 it.

**Review by annikaschmid — APPROVED** [review]
